### ISSUE TITLE
Trunking after the $ symbol in a password.

### ISSUE THREAD

**Issue by lsmanrique** [maxpozdeev/mytinytodo#40]
Trying to update an installation I got into a connection problem to the MySQL DB.

My hosting company, www.icdsoft.com has a Suresupport.com service which has helped me troubleshoot the problem. In their own words:

"Basically, it appears that the script strips the symbols after the dollar sign character ($) "

Kudos to them for this troubleshooting!  The updated webpage is now working flawlessly.

**Comment by maxpozdeev** [comment]
$ symbol in database password?

**Comment by maxpozdeev** [comment]
What mytinytodo version did you update and what is installed now?

**Comment by lsmanrique** [comment]
I went from 1.4 to 1.7

$ symbol was in my database password for MySQL, yes. I did not touch that data in config.php and tried an update (If I understood well, that means replacing all files except the folder /db/)  that failed with SQLSTATE[HY000] [1045] Access denied for user.  The support colleagues edited the password removing several characters in it, giving the origin of the connection problem to the dollar symbol. The original password had $%@#& symbols in it.
